lib/file/getFileBlobAsync.js

```diff
@@ -4,8 +4,10 @@ import { ENCODING, FILE_EXTENSION } from './constants';
 
 const ENCODING_TO_BLOB_GETTER = {
   [ENCODING.ASCII]: fileContent => Cypress.Promise.resolve(fileContent),
-  [ENCODING.BASE64]: (fileContent, mimeType) => Cypress.Blob.base64StringToBlob(fileContent, mimeType),
-  [ENCODING.BINARY]: (fileContent, mimeType) => Cypress.Blob.binaryStringToBlob(fileContent, mimeType),
+  [ENCODING.BASE64]: (fileContent, mimeType) =>
+    Cypress.Promise.try(Cypress.Blob.base64StringToBlob(fileContent, mimeType)),
+  [ENCODING.BINARY]: (fileContent, mimeType) =>
+    Cypress.Promise.try(Cypress.Blob.binaryStringToBlob(fileContent, mimeType)),
```

Comment:
```suggestion
    Cypress.Promise.try(() => Cypress.Blob.base64StringToBlob(fileContent, mimeType)),
  [ENCODING.BINARY]: (fileContent, mimeType) =>
    Cypress.Promise.try(() => Cypress.Blob.binaryStringToBlob(fileContent, mimeType)),
```

Adding the `() =>` is necessary for this to work.

Reply:
Yeah, `() =>` missing - however the code still does not work... :(